opentelemetry-sdk/src/opentelemetry/sdk/trace/samplers.py

```diff
@@ -0,0 +1,91 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from logging import getLogger
+from typing import Callable
+
+from opentelemetry.configuration import Configuration
+from opentelemetry.sdk.trace import sampling
+
+_logger = getLogger(__name__)
+
+
+class _Samplers:
+
+    registry = {}
```

Comment:
Would it be simpler to just have a hardcoded name -> class mapping instead of a registry?

Reply:
I think both are relatively same in terms of simplicity; having a registry is little more cosmetic than simple map.